scripts/apps/dashboard/views/workspace.html

```diff
@@ -64,7 +64,7 @@ <h3 class="content-state__heading" translate>Dashboard is empty</h3>
     </div>
 </div>
 
-<div sd-modal data-model="dashboard.add" class="modal--large modal--x-large modal--tabs">
+<div sd-modal data-model="dashboard.add" class="modal--large modal--x-large modal--tabs" data-test-id="'modal-widget'">
```

Comment:
Is it a widget displayed in a modal or a modal displayed in a widget? `widget-modal` would be a better name.